### ISSUE TITLE
Poloniex switched to new API on September 25th

### ISSUE THREAD

**Issue by DTM777** [ccxt/ccxt#3912]
ATTENTION!!!
     Poloniex switched their API to API V.2 on the 25th.

New api url follows:  

       wss://api2.poloniex.com

Thank you

**Comment by kroitor** [comment]
I suggest to move this discussion to #56 as it is related to WS implementation.

**Comment by DTM777** [comment]
First off,,  sorry for just shoutting help! :-0  :-(

Secondly: I really need to thank you and the hard working volunters that dedicate time and effort to this project.  I have learned much the 1st  version i did was in PHP, then tried to do one in JS and lastly python. mostly useing it to populate a MySql DB and inserting into professional stock charting software. I was able to get some cool charts working using matplotlib in python but really didn't want to reinvent the wheel to much. My very first attempt before I found your exelant program was a nightmare using excell and power query for ohlcv data that I then exported into the same aformentioned charting software

Thirdly: looks like ws / wss implementations of exchanges, is going to be a bit of a journey.  I read through that thread you linked this post to and all the links in each post on that page.  The WS is a bit over my head for the time being, but no-one learned what thay already knew. 
.... anyway ill stop rambleing, and begin reading docs.

Thank you;  Mr. Kroitor, and team.
